### PR TITLE
Update unified_service_tagging.md

### PR DESCRIPTION
**Pull request by lee-brian**
### Merge instructions

- [x] Please merge after reviewing

### CODE REVIEW

File: content/en/getting_started/tagging/unified_service_tagging.md [hunk 1]
```diff
@@ -72,6 +72,17 @@ To setup unified service tagging in a containerized environment:
 
 3. Configure your environment that corresponds to your container orchestration service based on either full configuration or partial configuration as detailed below.
 
+#### Automatic version tagging for containerized environments
```
Comment (yoannrobin) [inline content/en/getting_started/tagging/unified_service_tagging.md:75]: Any reason to not include the information in the Kubernetes/Docker/ECS section and add that DD_ENV is optional in these env?

File: content/en/getting_started/tagging/unified_service_tagging.md [hunk 2]
```diff
@@ -72,6 +72,17 @@ To setup unified service tagging in a containerized environment:
...
+You can use the `version` tag to [monitor deployments][17] and to identify faulty code deployments through [Automatic Faulty Deployment Detection][16].
+
+Datadog sets the `version` tag for you if the following criteria are met:
```
Comment (yoannrobin) [inline content/en/getting_started/tagging/unified_service_tagging.md:78]: it's a hard to parse. What do you think about adding a simple table with the order of priority and the value used?

**Review by yoannrobin — CHANGES_REQUESTED** [review]

File: content/en/getting_started/tagging/unified_service_tagging.md [hunk 3]
```diff
@@ -72,6 +72,17 @@ To setup unified service tagging in a containerized environment:
...
+You can use the `version` tag to [monitor deployments][17] and to identify faulty code deployments through [Automatic Faulty Deployment Detection][16].
+
+Datadog sets the `version` tag for you if the following criteria are met:  
+1. You do not set version yourself. If `version` is set, Datadog does not override your `version` value.
+2. Your service runs in a containerized environment, or you have enabled Git in the Datadog tracer. If you want to learn how to enable Git in the tracer, read the [Embed Git information in your build artifacts][18] documentation. 
+
+When those criteria are met, Datadog can set the`version` automatically:
+1. If both `{image_tag} and {git_commit_sha}` are available, `version` is set to {image_tag}_{git_commit_sha}.
+2. If only one tag is available then that tag's value is be used. For example: 'version' is set to only `{image_tag}` or `{git_commit_sha}`. 
+
```
Comment (yoannrobin) [inline content/en/getting_started/tagging/unified_service_tagging.md:85]: We also need to give the information on the minimum version where version can be automatically filled

File: content/en/getting_started/tagging/unified_service_tagging.md [hunk 4]
```diff
@@ -75,11 +75,122 @@ To setup unified service tagging in a containerized environment:
...
 
+##### Automatic version tagging for containerized environments
+You can use the `version` tag to [monitor deployments][17] and to identify faulty code deployments through [Automatic Faulty Deployment Detection][18].
+
+Datadog sets the `version` tag for you in the following priority order. If you manually set `version`, Datadog does not override your `version` value.
+
+| Priority         | Version Value |
+|--------------|------------|
+| 1    |  {your version value}       |
+| 2   | {image_tag}_{git_commit_sha}       |
```
Comment (jdgumz) [inline content/en/getting_started/tagging/unified_service_tagging.md:86]: Do we want to clarify that it's only the first 7 digits of the SHA?

File: content/en/getting_started/tagging/unified_service_tagging.md [hunk 5]
```diff
@@ -75,11 +75,122 @@ To setup unified service tagging in a containerized environment:
...
+| 3         |  {image_tag} or {git_commit_sha} if only one is available      |
+
+You need to install Datadog Agent Version 7.52.0 or greater and enable Git in the tracer to fully enable automatic version tagging. You can learn how to enable Git in the tracer by reading [Embed Git information in your build artifacts][19]
```
Comment (jdgumz) [inline content/en/getting_started/tagging/unified_service_tagging.md:89]: Maybe we can spell this out more clearly? The git integration is recommended but not absolutely required.

Requirements:
- Datadog Agent Version 7.52.0
- If running containerized and `image_tag` suffices, no further configuration necessary
- If not running containerized or if you'd also like to have the git SHA included, (embed Git information...)

Then again the simplest direction might be best.

**Review by jdgumz — CHANGES_REQUESTED** [review]

File: content/en/getting_started/tagging/unified_service_tagging.md [hunk 6]
```diff
@@ -187,16 +298,62 @@ containers:
...
 [3]: /agent/kubernetes/data_collected/#kube-state-metrics
 [4]: https://github.com/DataDog/integrations-core/blob/master/kubernetes_state/datadog_checks/kubernetes_state/data/conf.yaml.example
 [5]: /tracing/send_traces/
 [6]: /integrations/statsd/
+
 {{% /tab %}}
 
 {{% tab "Docker" %}}
+
+{{< collapse-content title="Set up with automatic version tagging" level="h4" >}}
```
Comment (yoannrobin) [inline content/en/getting_started/tagging/unified_service_tagging.md:314]: Is there a way to make this opened by default when clicking on the Docker tab? It is strange to have to make a choice between automatic and manual version tagging without recommendation

File: content/en/getting_started/tagging/unified_service_tagging.md (hunk 5 above)
Comment (lee-brian) [reply content/en/getting_started/tagging/unified_service_tagging.md:89]: Updated

File: content/en/getting_started/tagging/unified_service_tagging.md (hunk 6 above)
Comment (lee-brian) [reply content/en/getting_started/tagging/unified_service_tagging.md:314]: I couldn't make it open by default, but I just took it out of the collapsable section to make sure the user can see it. The manual instructions are now collapsed

**Review by lee-brian — COMMENTED** [review]
Resolved

File: content/en/getting_started/tagging/unified_service_tagging.md (hunk 4 above)
Comment (lee-brian) [reply content/en/getting_started/tagging/unified_service_tagging.md]: Updated

File: content/en/getting_started/tagging/unified_service_tagging.md [hunk 7]
```diff
@@ -75,11 +75,126 @@ To setup unified service tagging in a containerized environment:
...
+
+| Priority         | Version Value |
+|--------------|------------|
+| 1    |  {your version value}       |
+| 2   | {image_tag}_{first_7_digits_of_git_commit_sha}       |
+| 3         |  {image_tag} or {first_7_digits_of_git_commit_sha} if only one is available      |
+
+Requirements: 
+- Datadog Agent Version 7.52.0 or greater
+- If running containerized and `image_tag` suffices, no further configuration necessary
```
Comment (jdgumz) [inline content/en/getting_started/tagging/unified_service_tagging.md:91]: Ah you used my literal wording 😆 

Maybe
> If your services run in a containerized environment and `image_tag` is sufficient for tracking new version deployments, no further configuration is needed

And then
> If your services are not running in a containerized environment, or if you'd also like to have ... (same as it is now)

File: content/en/getting_started/tagging/unified_service_tagging.md [hunk 8]
```diff
@@ -240,10 +398,50 @@ com.datadoghq.tags.version
...
+[1]: /tracing/services/deployment_tracking/
+[2]: /watchdog/faulty_deployment_detection/
+[3]: /integrations/guide/source-code-integration/?tab=go#embed-git-information-in-your-build-artifacts
 
 {{% /tab %}}
 
 {{% tab "ECS" %}}
+
+#### Set up with automatic version tagging
+##### Full configuration
```
Comment (jdgumz) [inline content/en/getting_started/tagging/unified_service_tagging.md:412]: `Set up automatic version tagging` is the section but I don't see anything about version below? Is this an oversight?

**Review by jdgumz — APPROVED** [review]
LGTM on making some suggested changes / looking into a question I left.

**Review by knusbaum — APPROVED** [review]

File: content/en/getting_started/tagging/unified_service_tagging.md (hunk 7 above)
Comment (lee-brian) [reply content/en/getting_started/tagging/unified_service_tagging.md:96]: Updated

File: content/en/getting_started/tagging/unified_service_tagging.md (hunk 8 above)
Comment (lee-brian) [reply content/en/getting_started/tagging/unified_service_tagging.md]: I changed the structure so this doesn't apply anymore

File: content/en/getting_started/tagging/unified_service_tagging.md [hunk 9]
```diff
@@ -80,6 +80,22 @@ To setup unified service tagging in a containerized environment:
...
+| Priority         | Version Value |
+|--------------|------------|
+| 1    |  {your version value}       |
+| 2   | {image_tag}_{first_7_digits_of_git_commit_sha}       |
+| 3         |  {image_tag} or {first_7_digits_of_git_commit_sha} if only one is available      |
+
+Requirements: 
+- Datadog Agent Version 7.52.0 or greater
+- If running containerized and `image_tag` suffices, no further configuration necessary
+- If not running containerized or if you'd also like to have the git SHA included, [embed Git information in your build artifacts][9]
```
Comment (estherk15) [inline content/en/getting_started/tagging/unified_service_tagging.md:97]:
```suggestion
- Datadog Agent Version 7.52.0 or greater
- If your services run in a containerized environment and `image_tag` is sufficient for tracking new version deployments, no further configuration is needed
- If your services are not running in a containerized environment, or if you'd also like to have the git SHA included, [embed Git information in your build artifacts][9] 
```

File: content/en/getting_started/tagging/unified_service_tagging.md [hunk 10]
```diff
@@ -222,31 +258,53 @@ You may also prefer to set everything at deploy time:
...
+| Priority         | Version Value |
+|--------------|------------|
+| 1    |  {your version value}       |
+| 2   | {image_tag}_{first_7_digits_of_git_commit_sha}       |
+| 3         |  {image_tag} or {first_7_digits_of_git_commit_sha} if only one is available      |
+
+Requirements: 
+- Datadog Agent Version 7.52.0 or greater
+- If running containerized and `image_tag` suffices, no further configuration necessary
+- If not running containerized or if you'd also like to have the git SHA included, [embed Git information in your build artifacts][3]
```
Comment (estherk15) [inline content/en/getting_started/tagging/unified_service_tagging.md:303]:
```suggestion
- If your services run in a containerized environment and `image_tag` is sufficient for tracking new version deployments, no further configuration is needed
- If your services are not running in a containerized environment, or if you'd also like to have the git SHA included, [embed Git information in your build artifacts][3] 
```

**Review by estherk15 — APPROVED** [review]

File: content/en/getting_started/tagging/unified_service_tagging.md [hunk 11]
```diff
@@ -222,31 +258,53 @@ You may also prefer to set everything at deploy time:
...
+
+Requirements: 
+- Datadog Agent Version 7.52.0 or greater
+- If your services run in a containerized environment and `image_tag` is sufficient for tracking new version deployments, no further configuration is needed
+- If your services are not running in a containerized environment, or if you'd also like to have the git SHA included, [embed Git information in your build artifacts][3] 
+
 ##### Full configuration
 
-Set the `DD_ENV`, `DD_SERVICE`, and `DD_VERSION` environment variables and corresponding Docker labels in the runtime environment of each service's container to get the full range of unified service tagging. For instance, you can set all of this configuration in one place through your ECS task definition:
+Set the `DD_ENV`, `DD_SERVICE`, and `DD_VERSION` (optional with automatic version tagging)  environment variables and corresponding Docker labels in the runtime environment of each service's container to get the full range of unified service tagging. For instance, you can set all of this configuration in one place through your ECS task definition:
```
Comment (estherk15) [inline content/en/getting_started/tagging/unified_service_tagging.md:307]:
```suggestion
Set the `DD_ENV`, `DD_SERVICE`, and `DD_VERSION` (optional with automatic version tagging) environment variables and corresponding Docker labels in the runtime environment of each service's container to get the full range of unified service tagging. For instance, you can set all of this configuration in one place through your ECS task definition:
```
